demos/src/deploy-solution-main.ts

```diff
@@ -131,15 +138,19 @@ export function deployAndDisplaySolution(
   srcAuthentication: common.UserSession,
   destAuthentication: common.UserSession,
   progressCallback: common.ISolutionProgressCallback,
-  enableItemReuse: boolean
+  enableItemReuse: boolean,
+  customParams: any
 ): Promise<string> {
   // Deploy a solution described by the supplied id
   const options: common.IDeploySolutionOptions = {
     jobId: templateSolutionId,
     progressCallback: progressCallback,
     consoleProgress: true,
     storageAuthentication: srcAuthentication,
-    enableItemReuse
+    enableItemReuse,
+    templateDictionary: {
+      params: JSON.parse(customParams)
```

Comment:
If a solution template is supplied and the Custom Params input box is left alone, "Deploying..." appears on the screen, the app hangs, and the debugger shows "Uncaught SyntaxError: Unexpected end of JSON input".